3-enrich/spark-enrich/README.md

```diff
@@ -17,20 +17,20 @@ The process can be run on [Amazon Elastic MapReduce] [emr] using the [EmrEtlRunn
 Assuming you already have SBT installed:
 
     $ git clone git://github.com/snowplow/snowplow.git
-    $ cd 3-enrich/scala-hadoop-enrich
+    $ cd 3-enrich/spark-enrich
     $ sbt assembly
 
 The 'fat jar' is now available as:
 
-    target/snowplow-hadoop-etl-x.x.x.jar
+    target/snowplow-spark-etl-x.x.x.jar
```

Comment:
`sbt assembly` produces `target/scala-2.11/snowplow-spark-enrich-x.x.x.jar` (rightly so). I think the name of the jar should be consistent with the project name (`spark-enrich`).